mne/utils.py

```diff
@@ -1369,7 +1369,7 @@ def _get_extra_data_path(home_dir=None):
     if home_dir is None:
         # this has been checked on OSX64, Linux64, and Win32
         if 'nt' == os.name.lower():
-            home_dir = os.getenv('APPDATA')
+            home_dir = os.getenv('USERPROFILE')
```

Comment:
This is a backward incompatible change, so we can't do it like this.

Maybe `os.getenv('APPDATA', os.getenv('USERPROFILE'))`? That way you wouldn't get an error.

Reply:
to prevent error - yes
but for consistency is better:
`home_dir = os.getenv('APPDATA') if op.isdir(op.join(os.getenv('APPDATA'),'.mne')) else os.getenv('USERPROFILE')`

Reply:
+1 from me (with accompanying documentation update that before 0.16 the default was APPDATA and now it's USERPROFILE)

Reply:
+1 too